frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -461,7 +474,7 @@ export const funnelLogic = kea<funnelLogicType>({
                     ...(props.refresh ? { refresh: true } : {}),
                     ...(from_dashboard ? { from_dashboard } : {}),
                     ...cleanedParams,
-                    funnel_window_days: conversionWindowInDays,
+                    funnel_window_days: conversionWindow.days,
```

Comment:
Note that this is the deprecated parameter, we're now using these: https://github.com/PostHog/posthog/issues/5367#issuecomment-889860866

The idea being: the conversion window is not limited to days anymore, but can be hours/weeks/ etc.

Not a blocker for this PR, but just a heads up, since you're creating the `conversionWindow` type here anyway.

Reply:
(so there's opportunity for a better name here, even if you're using the old parameter for now!)

Reply:
oh awesome, didn't see the context there but I'll update the API call to use the new parameters. Thanks for the heads up

Reply:
Just saw that the backend gives us the flexibility to define units from [minutes all the way up to weeks](https://github.com/PostHog/posthog/blob/master/posthog/models/filters/mixins/funnel.py#L100). Do we want to propagate that kind of resolution up to the user? Or is days and weeks good for now? I'm not too sure about the use case here but I feel like option paralysis could be a thing. CC'ing @paolodamico @clarkus for UX guidance

Reply:
I can't think of a good reason to diverge from the units we offer in other places. I think supporting the full range and setting a reasonable default would be good. We generally default to the last 7 days, so something based on days seems like a good start? Thoughts?

Reply:
Added hours, days, weeks, and months for now.